docker/autoware-universe/Dockerfile

```diff
@@ -42,6 +42,12 @@ RUN rm -rf \
   /etc/apt/sources.list.d/docker.list \
   /etc/apt/sources.list.d/nvidia-docker.list
 
+## Register Vulkan GPU vendors
+ADD "https://gitlab.com/nvidia/container-images/vulkan/raw/dc389b0445c788901fda1d85be96fd1cb9410164/nvidia_icd.json" /etc/vulkan/icd.d/nvidia_icd.json
+RUN chmod 644 /etc/vulkan/icd.d/nvidia_icd.json
+ADD "https://gitlab.com/nvidia/container-images/opengl/raw/ubuntu20.04/glvnd/runtime/10_nvidia.json" /etc/glvnd/egl_vendor.d/10_nvidia.json
+RUN chmod 644 /etc/glvnd/egl_vendor.d/10_nvidia.json
```

Comment:
Although they are not large files, using `curl` might be better?
https://www.ctl.io/developers/blog/post/dockerfile-add-vs-copy/

Reply:
I was not convinced by the article. In the case we are in where the item to download is not an archive, the process seems very straightforward with `ADD`.
But if you want me to change to `curl`, I have no problem doing it.

Reply:
Actually, I personally prefer using `RUN curl/wget` because we can test the commands manually in a container like this.

```dockerfile
RUN wget -O /etc/vulkan/icd.d/nvidia_icd.json "https://gitlab.com/nvidia/container-images/vulkan/raw/dc389b0445c788901fda1d85be96fd1cb9410164/nvidia_icd.json"
```

```bash
docker run --rm -it ghcr.io/autowarefoundation/autoware-universe:galactic-latest-cuda
wget -O /etc/vulkan/icd.d/nvidia_icd.json "https://gitlab.com/nvidia/container-images/vulkan/raw/dc389b0445c788901fda1d85be96fd1cb9410164/nvidia_icd.json"
```

But I think it's okay to use `ADD` for this usage because the files are so small.

We don't have to stick to small things that linters don't show any warning and there is no rationale.